### DISCUSSION TITLE
Error 400: Bad Request

### DISCUSSION THREAD

**Discussion by cv55** [reactome/ReactomeGSA#61] [Q&A]
Hello, 

i am trying to use reactomeGSA to perform pathway analysis on a seurat object. I have log-normalized the counts, but I get an error when applying the function **analyse_sc_clusters**.

Calculating average cluster expression...
Converting expression data to string... (This may take a moment)
Conversion complete
Submitting request to Reactome API...
Compressing request data...
No encoding supplied: defaulting to UTF-8.
Error in start_reactome_analysis(request = request, compress = compress,  : 
  Request failed (400 - Bad Request): The browser (or proxy) sent a request that this server could not understand.

Any help would be appreciated. 
Thanks!

**Comment by jgriss** [comment]
Hi @cv55 

Sorry about the late response!
This issue was caused by a large hardware failure at our previous hosting provider.

We now moved to a different provider and the service is back onoline and should be working normally.

Please let us know in case the error persists!
